### PR TITLE
 Adjust Craft Times for XL Armor and Suits

### PR DESCRIPTION
**Pull request by reed501**
#### Summary

SUMMARY: Balance "Adjust craft times for XL armor and suits"

#### Purpose of change

XL Armor has craft times all over the place. Some are twice as long or more, other as as much as twice as short. Making the same thing but slightly bigger should just involve slightly longer crafts.

#### Describe the solution

For the most part XL armor has 12.5% more volume. I made craft times 12.5% longer-ish. I tried to round the numbers because who wants to see 1 day 13 hours and 17 minutes? Not me.

#### Describe alternatives you've considered

Making some new feature that copies the size scaling for recipe scaling.
25% longer craft times.

#### Testing

None yet.

#### Additional context

The other part of this PR I haven't finished yet but the idea is that suit crafting times are comically low. Even if one were to be the most proficient armor crafter ever, and even if they were to _also_ have the best equipment possible to make armor they could not come close to finishing a set of steel plate armor in 9 hours. Even from a gameplay handwave these don't make sense. It takes 14 hours to craft light survivor body armor but only 8 hours to craft the full suit. Crafting the individual pieces of a set of armor is usually taking 10x longer than crafting them as a joined suit with often better stats. Instead it should take approximately the length of them all summed, as the extra joining time balances out with the speed boost of doing it together.

I understand that tailoring times need to be increased a lot but it's hard to do without making items useless. For example I've set plate armor to take 24 hours if all proficiencies are known, which means a new character would see the craft time in weeks. However in reality this craft is going to take weeks for someone _with_ all the proficiencies. And someone with none might as well just see "craft time: no". Maybe this is a good spot to implement required proficiencies?

### CODE REVIEW

**Comment by hjk321** [comment]
Hey, if this PR is not finished, could you please add [WIP] to the title?
And if it is finished, could you please remove the draft tag? Thanks!

**Comment by actual-nh** [comment]
Technically, the increase should probably be proportional to surface area, not volume (and thus would be less). One can justify adding the extra time in absolutely necessary fitting for unusual morphology (that's without any fitting for decreased encumbrance).

**Review by Maleclypse — APPROVED** [review]
It's at least a system for why things take the length of time that they take.   I'm not sure if you are still here so I'm going to remove WIP from this.

File: data/json/recipes/armor/feet.json [hunk 1]
```diff
@@ -95,8 +95,8 @@
     "activity_level": "LIGHT_EXERCISE",
     "copy-from": "boots_acidchitin",
     "using": [ [ "armor_acidchitin", 23 ], [ "strap_small", 6 ], [ "clasps", 2 ] ],
-    "qualities": [ { "id": "CUT_FINE", "level": 1 } ],
-    "time": "19 h"
+    "time": "18 h"
```
Comment (Maleclypse) [inline data/json/recipes/armor/feet.json:98]:
```suggestion
    "time": "18 h",
```

File: data/json/recipes/armor/feet.json [hunk 2]
```diff
@@ -123,8 +123,8 @@
     "type": "recipe",
     "activity_level": "LIGHT_EXERCISE",
     "copy-from": "boots_fsurvivor",
-    "using": [ [ "tailoring_kevlar_fabric", 6 ], [ "tailoring_nomex_patchwork", 11 ], [ "fastener_small", 2 ] ],
-    "time": "23 h"
+    "time": "22 h 30 m"
```
Comment (Maleclypse) [inline data/json/recipes/armor/feet.json:126]:
```suggestion
    "time": "22 h 30 m",
```

File: data/json/recipes/armor/legs.json [hunk 3]
```diff
@@ -370,8 +370,8 @@
     "result": "xl_legguard_metal",
     "type": "recipe",
     "copy-from": "legguard_metal",
-    "time": "2 h",
-    "using": [ [ "steel_tiny", 6 ], [ "strap_large", 3 ], [ "clasps", 6 ] ],
+    "time": "50 m",
+    "using": [ [ "steel_tiny", 6 ], [ "strap_large", 2 ], [ "clasps", 4 ] ]
```
Comment (Maleclypse) [inline data/json/recipes/armor/legs.json:374]:
```suggestion
    "using": [ [ "steel_tiny", 6 ], [ "strap_large", 2 ], [ "clasps", 4 ] ],
```

File: data/json/recipes/armor/feet.json [hunk 4]
```diff
@@ -342,8 +342,8 @@
     "activity_level": "LIGHT_EXERCISE",
     "copy-from": "boots_plate",
     "using": [ [ "blacksmithing_standard", 37 ], [ "steel_standard", 10 ], [ "tailoring_leather_small", 20 ] ],
+    "time": "9 h"
```
Comment (Maleclypse) [inline data/json/recipes/armor/feet.json:345]:
```suggestion
    "time": "9 h",
```

File: data/json/recipes/armor/hands.json [hunk 5]
```diff
@@ -477,8 +477,8 @@
     "result": "xl_gloves_plate",
     "type": "recipe",
     "copy-from": "gloves_plate",
-    "time": "10 h",
-    "using": [ [ "blacksmithing_standard", 30 ], [ "steel_standard", 9 ], [ "clasps", 2 ], [ "strap_small", 2 ] ],
+    "time": "7 h 50 m",
+    "using": [ [ "blacksmithing_standard", 30 ], [ "steel_standard", 9 ], [ "clasps", 2 ], [ "strap_small", 2 ] ]
```
Comment (Maleclypse) [inline data/json/recipes/armor/hands.json:481]:
```suggestion
    "using": [ [ "blacksmithing_standard", 30 ], [ "steel_standard", 9 ], [ "clasps", 2 ], [ "strap_small", 2 ] ],
```

File: data/json/recipes/armor/legs.json [hunk 6]
```diff
@@ -346,8 +346,8 @@
     "result": "xl_legguard_lightplate",
     "type": "recipe",
     "copy-from": "legguard_lightplate",
-    "time": "5 h",
-    "using": [ [ "blacksmithing_standard", 42 ], [ "steel_standard", 9 ], [ "strap_small", 5 ], [ "clasps", 5 ] ],
+    "time": "3 h 30 m",
+    "using": [ [ "blacksmithing_standard", 42 ], [ "steel_standard", 9 ], [ "strap_small", 4 ], [ "clasps", 4 ] ]
```
Comment (Maleclypse) [inline data/json/recipes/armor/legs.json:350]:
```suggestion
    "using": [ [ "blacksmithing_standard", 42 ], [ "steel_standard", 9 ], [ "strap_small", 4 ], [ "clasps", 4 ] ],
```

**Pull request merged**